examples/move_it_mirror.ini

```diff
@@ -62,3 +62,24 @@ topic = /1b/hrit-segment/0deg
 # prog = /path/to/xRITDecompress
 # After decompression, delete the original compressed file
 # delete = True
+
+[eumetcast-hrit-0deg_scp_per_server_topic]
+# client part, the incoming data, providers with topics
+providers = satmottag2:9011/topic/1 satmottag:9011/topic/2
+login = user
+# Destination where the data are saved locally
+destination = scp:///tmp/bar/
+# Topics listened for advertised incoming data from all providers. Optional.
+client_topic = /1b/hrit-segment/0deg
```

Comment:
I'm not clear on the difference between this and what is specified in `providers`

Reply:
`client_topic` can be considered as a global default topic to subscribe to. So some of the providers may have a specific topic, and for the others the global one would still be available.